resemble.js

```diff
@@ -367,30 +352,17 @@ URL: https://github.com/Huddle/Resemble.js
 			px[offset + 3] = data.a * pixelTransparency; //a
 		}
 
-		function getPixelInfo(data, offset, cacheSet){
-			var r;
-			var g;
-			var b;
-			var d;
-			var a;
-
-			r = data[offset];
-
-			if(typeof r !== 'undefined'){
-				g = data[offset+1];
-				b = data[offset+2];
-				a = data[offset+3];
-				d = {
-					r: r,
-					g: g,
-					b: b,
-					a: a
-				};
+		function getPixelInfo(dst, data, offset, cacheSet) {
+			if (data.length > offset) {
+				dst.r = data[offset];
+				dst.g = data[offset + 1];
+				dst.b = data[offset + 2];
+				dst.a = data[offset + 3];
 
-				return d;
-			} else {
-				return null;
+				return true;
```

Comment:
Shouldn't this return `dst`, otherwise `targetPix = getPixelInfo(data, offset, cacheSet);` in `isAntialiased` will return a boolean. That will cause troubles when `targetPix` will be used in different methods.